venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,718 @@
+                    ],
+                writers = [
+                    'aclweb.org/ACL/2022/Conference'
+                    ],
+                members = acl_submission.content['authorids']
+            )
+            
+            client.post_group(authors)
+        
+        # Create paperX/Conflicts
+        if not openreview.tools.get_group(client, 'aclweb.org/ACL/2022/Conference/Paper{number}/Conflicts'.format(number = number)):
+            arr_submission = client.get_note(ARR_submission_forum)
+            conf_id = arr_submission.invitation.rsplit('/', 2)[0]
+            arr_number = arr_submission.number
+            conflicts = openreview.Group(
+                id = 'aclweb.org/ACL/2022/Conference/Paper{number}/Conflicts'.format(number = number),
+                signatures = [
+                    'aclweb.org/ACL/2022/Conference'
+                    ],
+                signatories = [
+                    'aclweb.org/ACL/2022/Conference'
+                    ],
+                readers = [
+                    'aclweb.org/ACL/2022/Conference'
+                    ],
+                writers = [
+                    'aclweb.org/ACL/2022/Conference'
+                    ],
+                members = [
+                    '{conf_id}/Paper{number}/Reviewers'.format(conf_id = conf_id, number = arr_number),
+                    '{conf_id}/Paper{number}/Area_Chairs'.format(conf_id = conf_id, number = arr_number),
+                    'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
+                    ]
+            )
+            client.post_group(conflicts)
+            
+            # Migrate all reviews from the original ARR Submission 
+            # Get invitation for that month's reviews 
+            review_invitation_ARR = '{conf_id}/Official_Review'.format(conf_id = conf_id)
+            metareview_invitation_ARR = '{conf_id}/Meta_Review'.format(conf_id = conf_id)
+            # Get all reviews from the original ARR Submission 
+            ARR_reviews = openreview.tools.iterget_notes(client, invitation = review_invitation_ARR, replyto = ARR_submission_forum)
+            ARR_metareviews = openreview.tools.iterget_notes(client, invitation = metareview_invitation_ARR, replyto = ARR_submission_forum)
+            
+            # Iterate through each review and for each, create and post a new review 
+            for ARR_review in ARR_reviews: 
+                ACL_review = openreview.Note(
+                    forum = acl_submission.forum,
+                    replyto = acl_submission.forum,
+                    invitation = 'aclweb.org/ACL/2022/Conference/-/Official_Review',
+                    signatures = [
+                        'aclweb.org/ACL/2022/Conference'
+                    ],
+                    readers = [
+                        'aclweb.org/ACL/2022/Conference',
+                        'aclweb.org/ACL/2022/Conference/Program_Chairs'
```

Comment:
shouldn't the SACs be readers of the review?

Reply:
Oh yes they should, I will add that. Would I need to use values-copied for the review invitation reply parameters to include the correct SAC group?